ocf_data_sampler/torch_datasets/datasets/site.py

```diff
@@ -242,8 +242,9 @@ def process_and_combine_site_sample_dict(
         # add datetime features
         datetimes = pd.DatetimeIndex(combined_sample_dataset.site__time_utc.values)
         datetime_features = make_datetime_numpy_dict(datetimes=datetimes, key_prefix="site")
```

Comment:
Actually small thing but I think to keep the consistent format of `{key}__` at the start of all coordinates/dims in the dataset I think we would want 
```suggestion
        datetime_features = make_datetime_numpy_dict(datetimes=datetimes, key_prefix="site_")
```